recipes/configure.rb

```diff
@@ -89,7 +89,9 @@
 # The updating of the list of checks.
 ruby_block 'updating of the list of checks' do
   block do
-    checks = run_context.resource_collection.select { |r| r.is_a?(Chef::Resource::NrpeCheck) && r.action == [:add] }.map(&:command_name)
+    checks = run_context.resource_collection.select do |r|
+      r.is_a?(Chef::Resource::NrpeCheck) && r.action == [:add]
```

Comment:
r.action is een array, dus meestal doe ik iets van r.action.includes?(:add) zodat je ook een array van [:nothing, :add] meeneemt.